### PR TITLE
SpreadDamageWarhead Range on ruleset loaded

### PR DESCRIPTION
**Pull request by atlimit8**
Includes better error reporting.

### CODE REVIEW

File: OpenRA.Game/GameRules/Ruleset.cs [hunk 1]
```diff
@@ -45,7 +45,23 @@ public class Ruleset
 
 			foreach (var a in Actors.Values)
 				foreach (var t in a.TraitInfos<IRulesetLoaded>())
-					t.RulesetLoaded(this, a);
+					try {
```
Comment (pchote) [inline OpenRA.Game/GameRules/Ruleset.cs]: Please add braces to the outer loops, and then move these ones on to their own line to fit with our usual style conventions. (and the same again below).

**Comment by atlimit8** [comment]
Changed.

**Comment by pchote** [comment]
Looks good here too :+1:

**Comment by obrakmann** [comment]
:+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/13091924827e699443781fad209f6b3ff5ff57b8)

**Pull request merged**